Dockerfile.openshift

```diff
@@ -0,0 +1,16 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.11 AS builder
+ARG OS=linux
+ARG ARCH=amd64
+ARG GOVERSION=1.11.1
+WORKDIR /go/src/sigs.k8s.io/sig-storage-local-static-provisioner
+COPY . .
+RUN CGO_ENABLED=0 GOOS=${OS} GOARCH=${ARCH} go build -a -ldflags '-extldflags "-static"' -o _output/${OS}/${ARCH}/local-volume-provisioner ./cmd/local-volume-provisioner
+
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+COPY --from=builder /go/src/sigs.k8s.io/sig-storage-local-static-provisioner/_output/${OS}/${ARCH}/local-volume-provisioner /local-provisioner
+ADD deployment/docker/scripts /scripts
+ENTRYPOINT ["/local-provisioner"]
+LABEL io.openshift.release.operator true
+LABEL io.k8s.display-name="OpenShift local-storage-static-provisioner" \
+      io.k8s.description="This is a component of OpenShift and provisions local volumes." \
+        maintainer="Hemant Kumar <hekumar@redhat.com>"
```

Comment:
This Dockerfile does not actually work with podman because of https://bugzilla.redhat.com/show_bug.cgi?id=1717717